### PR TITLE
Add winter/sports icons/sort `hat-*` icons

### PR DESCRIPTION
**Pull request by danielbayley**
Loosely related to #1304.

`ski`/`snowboard` alts:

preview

`scarf` alts:

preview

### CODE REVIEW

File: icons/snowman.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+>
+  <path d="M2 9h2V7" />
+  <path d="M7 12 4 9" />
+  <path d="m18 11 2-2" />
+  <path d="M20 7v2h2" />
+  <circle cx="12" cy="6" r="4" />
+  <path d="M12 14h.01" />
+  <path d="M12 18h.01" />
+  <path d="M9 8.7a7 7 0 1 0 6 0" />
+  <path d="M5 22h14" />
```
Comment (jguddas) [inline icons/snowman.svg:20]: icon (92)

```suggestion
  <circle cx="12" cy="6" r="4" />
  <path d="M2 9h2V7" />
  <path d="M6 11.5 4 9" />
  <path d="M18 11.5 20 9" />
  <path d="M20 7v2h2" />
  <path d="M 12 14h0.01" />
  <path d="M 12 18h0.01" />
  <path d="M9 8.7a7 7 0 1 0 6 0" />
  <path d="M5 22h14" />
```

Comment (jguddas) [reply icons/snowman.svg:20]: With eye and nose: icon (93)

File: icons/skis.svg
Comment (karsa-mistmere) [inline icons/skis.svg:1]: Not super thrilled about these or any of the alts, the skis themselves look more like stakes or picket fence. 🤔

File: icons/snowboard.svg
Comment (karsa-mistmere) [inline icons/snowboard.svg:1]: Not sure how recognisable this one is. Why snowboard?
Could also easily be a skateboard, hoverboard or basically any type of pill or sausage shaped item.

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/medal.svg [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
 >
-  <path d="M7.21 15 2.66 7.14a2 2 0 0 1 .13-2.2L4.4 2.8A2 2 0 0 1 6 2h12a2 2 0 0 1 1.6.8l1.6 2.14a2 2 0 0 1 .14 2.2L16.79 15" />
-  <path d="M11 12 5.12 2.2" />
-  <path d="m13 12 5.88-9.8" />
+  <path d="M17 15s2.9-4.6 4.8-7.7a2 1.65 0 0 0-.1-1.6l-2-3c-.2-.4-.7-.7-1.2-.7h-13c-.5 0-1 .3-1.2.7l-2 3a2 1.65 0 0 0-.1 1.6C4.1 10.4 7 15 7 15" />
   <path d="M8 7h8" />
+  <path d="M11 12 5 2.1" />
+  <path d="m13 12 6-9.9" />
   <circle cx="12" cy="17" r="5" />
   <path d="M12 18v-2h-.5" />
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/medal.svg:17]: > why change this in the first place?

@karsa-mistmere

1. It has an irregular width of 19.2134 px
2. To try and better match `scarf` above

File: icons/scarf.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "danielbayley"
+  ],
+  "tags": [
+    "wooly",
+    "woolly",
+    "winter",
+    "cold",
+    "neckwear",
+    "apparel",
+    "fashion"
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/scarf.json]: @karsa-mistmere See also alts above: https://github.com/lucide-icons/lucide/pull/1435#issue-1806569455

File: icons/skis.svg
Comment (danielbayley) [reply icons/skis.svg:1]: @karsa-mistmere What about something like these…?

preview

File: icons/snowman.svg (hunk 1 above)
Comment (karsa-mistmere) [reply icons/snowman.svg]: I very much preferred the original design, this one looks more like a chubby little penguin than a snowman.

File: icons/medal.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/medal.svg]: > my bad, I mixed up the two when I copied them to Figma

> There are still some misaligned paths

@karsa-mistmere I think something similar might have happened again… If you check above https://github.com/lucide-icons/lucide/pull/1435#issuecomment-1637076021 I’m pretty sure the paths are as aligned as they can be now.

Also, why do modified icons now show up as red on red in the preview bot? Surely they should contrast better? @jguddas

File: icons/snowboard.svg
Comment (danielbayley) [reply icons/snowboard.svg:1]: Trying to go off `surfboard` in #1393 here… How about a slightly more ‘squeezed’ shape, or any other ideas for the pattern?

preview

File: icons/medal.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (karsa-mistmere) [reply icons/medal.svg]: Huh, weird, the old one seems to have been cached, I had to delete my browser cache before the correct version showed up. 👀 

There's still some minor issues with the horizontal line:
image

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Pull request merged**